src/main/webapp/scripts/navbar.js

```diff
@@ -37,6 +37,10 @@ function getActiveTab() {
 
 function setActiveTab(fileName) {
   const activeTabElement = document.getElementById(`${fileName}_tab`);
+  // If its Edit Card then return
+  if (activeTabElement === null) {
+    return;
+  }
```

Comment:
How does this work? 
Edit card does not have a `${fileName}_tab`?
Btw, I think you can write just `if (!activeTabElement)`.

Reply:
Now that I see it, the comment is wrong. I meant to write View Card, which doesn't have a tab on the navbar. `${filename}_tab` represents the active tab on the navbar.